db/schema.rb

```diff
@@ -13,47 +12,50 @@
 
 ActiveRecord::Schema.define(version: 20160421164630) do
 
-  create_table "entries", force: :cascade do |t|
-    t.text     "description"
-    t.integer  "minutes"
-    t.integer  "user_id"
-    t.integer  "project_id"
-    t.date     "date"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+  # These are extensions that must be enabled in order to support this database
```

Comment:
I think we shouldn't include these change, they are unrelated to the review apps stuff